src/gc/impl/conservative/gc.d

```diff
@@ -260,6 +294,7 @@ class ConservativeGC : GC
     import core.internal.spinlock;
     static gcLock = shared(AlignedSpinLock)(SpinLock.Contention.lengthy);
     static bool _inFinalizer;
+    static bool isPrecise=false;
```

Comment:
I'm a little fuzzy on the proper usage of `__gshared`. Should anything marked `__gshared` be in a module's global scope or can they still be a member of a struct/class?

>Long term, these statics should be members instead, probably by merging Gcx into ConservativeGC. Why are these separated to begin with?

I'm actually planning to work on that after GSoC is over.